### DISCUSSION TITLE
Unable to Rename GitHub Account Even Though There Isn't Existing Account

### DISCUSSION THREAD

**Discussion by everforyou** [community/community#138582] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Github Support Team,

I’m trying to change my GitHub account name from `rabbitbrother91` to `rabbitbrother`. However, even though there isn’t an existing account with the name `rabbitbrother`, I’m unable to use that username. 

I would appreciate any insights on the following:

- Why am I unable to use `rabbitbrother` if it appears to be available?
- Are there any specific steps or restrictions I should be aware of when renaming my account?
- Is there a way to contact GitHub support for clarification on this issue?

Please help me to rename with it must.
Thank you for your help!

**Comment by davevad93** [comment, marked as answer]
Hi @rabbitbrother91 ,

For account related issues it's better to ask directly to support, raise a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem about the username change.

**Reply by everforyou** [reply]
Thanks @davevad93.

**Comment by ebndev** [comment]
Hi @rabbitbrother91,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

You may not be seeing the username as it could belong to a private entity or was previously claimed but now deleted.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!

**Reply by everforyou** [reply]
Hi @ebndev,

Could you please assist me in changing my username on GitHub at https://github.com/rabbitbrother, if possible?
![image](https://github.com/user-attachments/assets/82dc9870-c080-4f59-b42f-9c476d8c88a6)

Thanks in advance
